test.js

```diff
@@ -1,148 +1,203 @@
-        outplan.create("foo", ["A", "B"]);
-        var value = outplan.expose("foo", 1);
-        assert(value === "A" || value === "B", value);
-    });
-    
-    it("deterministically gets values", function() {
-        outplan.create("foo", ["A", "B"]);
-        assert.equal(outplan.expose("foo", 1), "B");
-        assert.equal(outplan.expose("foo", 2), "B");
-        assert.equal(outplan.expose("foo", 3), "A");
-        assert.equal(outplan.expose("foo", 4), "B");
-        assert.equal(outplan.expose("foo", "1"), "B");
-        assert.equal(outplan.expose("foo", "2"), "B");
-        assert.equal(outplan.expose("foo", "3"), "A");
-        assert.equal(outplan.expose("foo", "4"), "B");
-        assert.equal(outplan.expose("foo", "5"), "B");
-        assert.equal(outplan.expose("foo", "6"), "A");
-        assert.equal(outplan.expose("foo", "7"), "A");
-        assert.equal(outplan.expose("foo", "8"), "B");
-        assert.equal(outplan.expose("foo", "9"), "A");
+        testBoth(function(outplan) {
+            outplan.create("foo", ["A", "B"]);
+            var value = outplan.expose("foo", 1);
+            assert(value === "A" || value === "B", value);
+        });
+    });
+    
+    it("deterministically gets values in outplan_full", function() {
+        outplanFull.create("foo", ["A", "B"]);
+        assert.equal(outplanFull.expose("foo", 1), "B");
+        assert.equal(outplanFull.expose("foo", 2), "B");
+        assert.equal(outplanFull.expose("foo", 3), "A");
+        assert.equal(outplanFull.expose("foo", 4), "B");
+        assert.equal(outplanFull.expose("foo", "1"), "B");
+        assert.equal(outplanFull.expose("foo", "2"), "B");
+        assert.equal(outplanFull.expose("foo", "3"), "A");
+        assert.equal(outplanFull.expose("foo", "4"), "B");
+        assert.equal(outplanFull.expose("foo", "5"), "B");
+        assert.equal(outplanFull.expose("foo", "6"), "A");
+        assert.equal(outplanFull.expose("foo", "7"), "A");
+        assert.equal(outplanFull.expose("foo", "8"), "B");
+        assert.equal(outplanFull.expose("foo", "9"), "A");
+    });
+    
+    it("deterministically gets values in lite version", function() {
+        outplanLite.create("foo", ["A", "B"]);
+        assert.equal(outplanLite.expose("foo", 1), "A");
+        assert.equal(outplanLite.expose("foo", 2), "B");
+        assert.equal(outplanLite.expose("foo", 3), "A");
+        assert.equal(outplanLite.expose("foo", 4), "B");
+        assert.equal(outplanLite.expose("foo", "1"), "A");
+        assert.equal(outplanLite.expose("foo", "2"), "B");
+        assert.equal(outplanLite.expose("foo", "3"), "A");
+        assert.equal(outplanLite.expose("foo", "4"), "B");
+        assert.equal(outplanLite.expose("foo", "5"), "A");
+        assert.equal(outplanLite.expose("foo", "6"), "B");
+        assert.equal(outplanLite.expose("foo", "7"), "A");
+        assert.equal(outplanLite.expose("foo", "8"), "B");
+        assert.equal(outplanLite.expose("foo", "9"), "A");
```

Comment:
For reference I'm using https://github.com/cloud9ide/outplan/pull/1/files#diff-50a581e412c9818e1931179309559048R4 for hashing here. We need to have a better hasher.